tutorials/pulsar_analysis.ipynb

```diff
@@ -0,0 +1,464 @@
+    "mask_off = (0.7 < phases) & (phases < 1)  # mask for the off-phase zone\n",
+    "count_bkg = mask_off.sum()  # number of phases in the off-phase zone\n",
+    "bkg = count_bkg / nbins / 0.3  # bkg lebel normalized by the size of the OFF zone (0.3)\n",
+    "\n",
+    "# Plot background level\n",
+    "x_bkg = np.linspace(0, 1, 50)\n",
+    "plt.plot(x_bkg, bkg*np.ones(len(x_bkg)), color='black', alpha=.5, ls='--', lw=2)\n",
+    "\n",
+    "# Let's make patches for the on and off phase zones\n",
+    "on_patch = plt.axvspan(0.5, 0.6, alpha=0.3, color='gray')\n",
+    "pylab.setp(on_patch, ec='black', lw=1, ls='solid')\n",
+    "off_patch=plt.axvspan(0.7, 1.0, alpha=0.4, color='white', hatch='x')\n",
+    "pylab.setp(off_patch, ec='black', lw=1, ls='solid')\n",
+    "\n",
+    "# Legends \"ON\" and \"OFF\"\n",
+    "plt.text(0.55, 5, 'ON', color = 'black',fontsize= 17, ha='center')\n",
+    "plt.text(0.895, 5, 'OFF', color = 'black',fontsize= 17, ha='center')\n",
+    "\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## III : Phase-resolved map"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now that the phases are computed, we want to do a phase-resolved sky map : a map of the ON-phase events minus alpha times the OFF-phase events."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from gammapy.maps import Map\n",
+    "from gammapy.cube import fill_map_counts, MapMaker\n",
+    "\n",
+    "\n",
+    "# Let's create an ON-map and an OFF-map\n",
+    "on_map = Map.create(binsz=0.1, map_type='wcs', skydir=pos_target, width=10.0)\n",
+    "off_map = Map.create(binsz=0.1, map_type='wcs', skydir=pos_target, width=10.0)\n",
+    "\n",
+    "# Loop to fill the ON- and OFF- maps\n",
+    "on_mask = np.where(np.logical_and(0.5 < phases, 0.6 > phases))[0]\n",
+    "fill_map_counts(on_map, one_obs_vela_select.select_row_subset(on_mask))\n",
+    "off_mask = np.where(np.logical_and(0.7 < phases, 1 > phases))[0]\n",
+    "fill_map_counts(off_map, one_obs_vela_select.select_row_subset(off_mask))\n",
+    "\n",
+    "# Create and fill excess map\n",
+    "excess_map = Map.create(binsz=0.1, map_type='wcs', skydir=pos_target, width=10.0)\n",
+    "excess_map.data = on_map.data - 0.1/0.3 * off_map.data\n",
+    "\n",
+    "# Plot excess map\n",
+    "excess_map.plot()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# We can smooth it\n",
+    "\n",
+    "excess_map.smooth(kernel='gauss', width=0.2 * u.deg).plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## IV : Phase-resolved spectrum"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can also do a phase-resolved spectrum."
```

Comment:
Here you really need to explain what is going to happen both for analysis and code.